orc8r/tools/packer/vagrant-box-upload.sh

```diff
@@ -3,6 +3,7 @@
 set -euo pipefail
 
 BOX_FILE=$1
+FORCE="--no-force"
```

Comment:
_optional:_ Might this be better collected before the respective block? (Very opinionated.)